src/js/edu-benefits/reducers/veteran/index.js

```diff
@@ -1,14 +1,27 @@
 import _ from 'lodash/fp';
 
 import { VETERAN_FIELD_UPDATE, ENSURE_FIELDS_INITIALIZED } from '../../actions';
-import { makeField } from '../../../common/model/fields';
+import { makeField, dirtyAllFields } from '../../../common/model/fields';
 
 const blankVeteran = {
   benefitsRelinquished: makeField(''),
   chapter30: false,
   chapter1606: false,
   chapter32: false,
-  chapter33: false
+  chapter33: false,
+  veteranFullName: {
```

Comment:
Good point. I was trying to make this consistent with the names HCA used. Perhaps the schema for education should change then so it's all consistent?